README.md

```diff
@@ -15,13 +15,7 @@ Thanks for this opportunity.
 
 Bonsoir les Chats 
 
-## :sparkles: Hey Kottans! :sparkles:
 
-[I'm](https://github.com/maxpetruk) delighted to join the community! 
```

Comment:
Ouch, now you deleted all your addings and even deleted someone's else greeting?
Don't worry about conflicts - I can help with that, but you can't delete someone's else work; there should not be those red lines.

Reply:
Sure, but I am not sure how to edit back.

Reply:
I tried the git revert command but it is not working.
I typed: "git revert"